packages/atomic/src/components/facets-v1/facet-search/facet-search-input.tsx

```diff
@@ -25,15 +25,14 @@ export const FacetSearchInput: FunctionalComponent<FacetSearchInputProps> = (
       <div class="relative h-10">
         <input
           part="search-input"
-          class="w-full h-full border border-neutral bg-background rounded px-9 placeholder-neutral-dark text-sm hover:border-primary-light focus:border-primary focus:outline-none group"
+          class="input-primary w-full h-full px-9 placeholder-neutral-dark text-sm group"
           type="text"
           placeholder={search}
           aria-label={facetSearch}
           value={props.query}
           onInput={(e) => props.onChange((e.target as HTMLInputElement).value)}
           ref={(ref) => (inputRef = ref)}
         />
-        <div class="input-ring invisible ring-2 w-full h-full absolute inset-0 pointer-events-none rounded opacity-40 ring-primary"></div>
```

Comment:
This was the previous "hack" to put a transparent ring around the search input using the primary color. IMO it is not super clean and is a pain to reuse the hack elsewhere, opted for a new color instead.